formats/qcow.nix

```
@@ -1,4 +1,4 @@
-{ config, lib, pkgs, modulesPath, ... }:
+{ config, lib, pkgs, modulesPath, diskSize ? 8192, ... }:
```

Would be "auto" a good default?

```suggestion
{ config, lib, pkgs, modulesPath, diskSize ? "auto", ... }:
```